c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -1458,6 +1458,11 @@ mfxStatus MfxC2DecoderComponent::HandleFormatChange()
         m_uMaxHeight = m_mfxVideoParams.mfx.FrameInfo.Height;
     }
 
+    if ((DECODER_H264 == m_decoderType || DECODER_H265 == m_decoderType) && m_c2Bitstream) {
```

Comment:
I suspect other codecs also need this bitstream reset operation when format change happened...

Thanks,
Tianmi

Reply:
Hi Tianmi, at here, only AVC and HEVC need this operation. 
Because SPS and PPS are only defined in AVC and HEVC spec. 
For VP9 and AV1, parameters are defined in intra frame, instead of header.
the target of reset operation is for re-append SPS and PPS header for bitstream. it is only needed for AVC and HEVC